resources/lang/en-GB/people.php

```diff
@@ -308,7 +308,7 @@
     'gifts_delete_cta' => 'Delete',
     'gifts_add_title' => 'Gift management for :name',
     'gifts_add_gift_idea' => 'Gift idea',
-    'gifts_add_gift_already_offered' => 'Gift offered',
+    'gifts_add_gift_already_offered' => 'Gift given',
```

Comment:
Sorry but `en-GB` is also a translation, and must not be edited (see the header of the file).

Reply:
Okay. I’ve made a commit that _only_ changes the `en` file, and pushed it.